Objective-C/CBLDatabase.mm

```diff
@@ -115,7 +114,8 @@ to be done in CBLInit()which is called only once when the first CBLDatabase is c
  */
 + (void) initialize {
     if (self == [CBLDatabase class]) {
-        NSLog(@"%@", [CBLVersion userAgent]);
+        os_log_t logger = os_log_create("com.couchbase.lite.ios", "UserAgent");
+        os_log(logger, "%@", [CBLVersion userAgent]);
```

Comment:
I think you can add a static method at CBLConsoleLog to log the message regardless of the log level so that there is only one place that manages the logger? The category could be just "Database" as well.